src/Runner.js

```diff
@@ -298,6 +298,10 @@ function run(transformFile, paths, options) {
             process.stdout.write(
               'Time elapsed: ' + timeElapsed + 'seconds \n'
             );
+            
+            if (options.failOnError && fileCounters.error > 0) {
+              process.exit(1);
```

Comment:
This should probably go below the `temp.cleanupSync` call at least?

Is it possible to put the `process.exit(1)` call in `bin/jscodeshift.js` rather than in this file? I suspect this file is supposed to be reusable as a library and thus shouldn't be handling the exit here.

Reply:
Good point on moving the process.exit(1), it also didn't really belong in the `if (!options.silent) {` statement.

On moving this to the bin, I think it makes sense to have it in this file since Runner.js already does many process stuff (see also the process.stdout). If you want to use this file in other JS code, the user can not pass in the failOnError flag.

But it looks like it passes back the stats to bin/jscodeshift, so it's easy to move there as well, it was just that bin/jscodeshift seemed like just the cli wrapper